$NetBSD: patch-src_Makevars.in,v 1.2 2026/07/04 23:17:05 mef Exp $

 Undefined PLT symbol "backtrace_symbols"

 this symbol found at /usr/lib/libexecinfo for *BSD

--- src/Makevars.in.orig	2023-09-07 03:40:05.000000000 +0900
+++ src/Makevars.in	2026-07-05 08:10:34.026713533 +0900
@@ -1,2 +1,4 @@
 PKG_CPPFLAGS = -DUDUNITS2_DIR=@UDUNITS2_DIR@ @UD_CPPFLAGS@
 PKG_LIBS = @LIBS@
+
+@@PKG_LIBS@@
